File: backend/src/main/java/au/com/anthonybruno/gymbuddy/auth/UserCredentials.java
package au.com.anthonybruno.gymbuddy.auth;

import com.fasterxml.jackson.annotation.JsonProperty;

public class UserCredentials {

    private final String username;
    private final String password;

    public UserCredentials(@JsonProperty("username") String username, @JsonProperty("password") String password) {
        this.username = username;
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public String getPassword() {
        return password;
    }
}

File: backend/src/main/kotlin/dev/anthonybruno/gymbuddy/auth/UserCredentials.kt
package dev.anthonybruno.gymbuddy.auth;

import com.fasterxml.jackson.annotation.JsonProperty;

public class UserCredentials {

    private final String email;
    private final String password;

    public UserCredentials(@JsonProperty("email") String email, @JsonProperty("password") String password) {
        this.email = email;
        this.password = password;
    }

    public String getEmail() {
        return email;
    }

    public String getPassword() {
        return password;
    }
}
